apps/charterafrica/src/lib/data/fetchApi.js

```diff
@@ -0,0 +1,68 @@
+import qs from "qs";
+
+export class FetchApi {
+  headers = {
+    "Content-Type": "application/json",
+  };
+
+  token = "";
+
+  email = process.env.NEXT_PUBLIC_APP_AUTH_EMAIL;
+
+  password = process.env.NEXT_PUBLIC_APP_AUTH_PASSWORD;
+
+  constructor(headers) {
+    if (headers) {
+      this.headers = headers;
+      return;
+    }
+    this.authenticate();
+  }
+
+  async authenticate() {
+    const data = { email: this.email, password: this.password };
+    const res = await this.post(
+      `${process.env.NEXT_PUBLIC_APP_URL}/api/users/login`,
```

Comment:
While they'll point to the same thing, any call meant to go to the Payload CMS should use `PAYLOAD_PUBLIC_APP_URL`.